specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/examples/createOrUpdateDeployment.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "inferenceEndpointName": "testendpoint",
+    "deploymentName": "testdeployment",
+    "api-version": "2020-09-01-preview",
+    "properties": {
+      "modelId": [
```

Comment:
Should this be `modelIds` instead of `modelId`? ModelValidation is reporting this. Please double check